### ISSUE TITLE
Trace logger doesn't get along with some symbolic debug

### ISSUE THREAD

**Issue by romhacking** [TASVideos/fceux#30]
If I make a comment right after RTS line in debugger, and its length is 28 or more symbols in "name" field, trace logger really hates it. Took me a while to figure out what's wrong with my .nl files.

Here is a video with demonstration https://youtu.be/8FUcUsp20NU

Tested on current a84ccfe version.

**Comment by romhacking** [comment]
Also, why "name" field can only contain 32 symbols? It is really inconvenient. Can you make it fit more, a 100 maybe?

**Comment by zeromus** [comment]
Also, change `#define NL_MAX_NAME_LEN 30` in debuggersp.h and see if anything explodes

**Comment by romhacking** [comment]
Thank you for the fix.

I will test more symbols when I get that fix about "pause" hotkey, it's is really inconvenient for me to work without it.

**Comment by romhacking** [comment]
Ok, I've changed it to "1000" and been using it for a month, haven't noticed any explosions :) Please add this to your next update.

**Comment by romhacking** [comment]
Unfortunately, yesterday I have found a weird side effect from this commit https://github.com/TASVideos/fceux/commit/cb5ff1bcea2e9e86f50986f1aa9f72cdf5522ad5

Watch this video and prepare for annoying sounds https://youtu.be/DhtWQJU64U8

Steps from video:

First I launched builded exe from solution without debugger in VS
1 - 30 symbols in "Name" field, everything is fine
2 - 37 symbols, sound is played after breakpoint hit
3 - 59 symbols, sound + background dissapears, can be fixed only by restarting emulator
4 - 68 symbols, emulator crashes when I run the game and press turbo button

Then I launched fceux with debugger in VS. No sound played and background doesn't turns off here, however with 68 symbols it crashes imidiately after running a game.

This doesn't happen with all breakpoint's hits, this is the first time I see it, maybe it is a particular case or something.

[files for testing.zip](https://github.com/TASVideos/fceux/files/3262329/files.for.testing.zip)
